Assets/HoloToolkit-Preview/SpectatorView/Scripts/SpatialSync/MarkerDetector.cs

```diff
@@ -6,29 +6,29 @@
 using System.Runtime.InteropServices;
 using UnityEngine;
 
-namespace HoloToolkit.Unity.SpectatorView
+namespace HoloToolkit.Unity.Preview.SpectatorView
 {
     /// <summary>
     /// Manages the OpenCV wrapper to detect a marker.
     /// </summary>
     public class MarkerDetector
     {
-        [DllImport("SpectatorViewPlugin", EntryPoint="MarkerDetector_Initialize", ExactSpelling=true)]
+        [DllImport("SpectatorViewPlugin", EntryPoint="MarkerDetector_Initialize")]
```

Comment:
Do we want exact spelling? 

Or was this part of the problem we had with building and compiling?

Reply:
I got reports that that was blocking the runtime from finding the entry points. ExactSpelling quiets the compiler warnings, but apparently ignores the EntryPoint param in the attribute.